Source/Core/Core/HW/WII_IPC.cpp

```diff
@@ -170,6 +232,196 @@ void Shutdown()
 {
 }
 
+static std::string_view GetAVERegisterName(u8 address)
+{
+  if (address == 0x00)
+    return "A/V Timings";
+  else if (address == 0x01)
+    return "Video Output configuration";
+  else if (address == 0x02)
+    return "Vertical blanking interval (VBI) control";
+  else if (address == 0x03)
+    return "Composite Video Trap Filter control";
+  else if (address == 0x04)
+    return "A/V output control";
+  else if (address == 0x05 || address == 0x06)
+    return "CGMS protection";
+  else if (address == 0x08 || address == 0x09)
+    return "WSS (Widescreen signaling)";
+  else if (address == 0x0A)
+    return "RGB color output control";
+  else if (address >= 0x10 && address <= 0x30)
+    return "Gamma coefficients";
+  else if (address >= 0x40 && address <= 0x59)
+    return "Macrovision code";
+  else if (address == 0x62)
+    return "RGB switch control";
+  else if (address == 0x65)
+    return "Color DAC control";
+  else if (address == 0x67)
+    return "Color Test";
+  else if (address == 0x6A)
+    return "CCSEL";
+  else if (address == 0x6D)
+    return "Audio mute control";
+  else if (address == 0x6E)
+    return "RGB output filter";
+  else if (address == 0x71)
+    return "Audio stereo output control - right volume";
+  else if (address == 0x72)
+    return "Audio stereo output control - right volume";
```

Comment:
Seems likethis was meant to be the left channel volume, otherwise it seems really weird that the right channel appears twice

Reply:
Yeah, probably it should be left, though the wiki is a bit ambiguous: https://wiibrew.org/wiki/Hardware/AV_Encoder#Registers_71h_.26_72h